### DISCUSSION TITLE
bufferline doubleclick callback? can this be a thing ?

### DISCUSSION THREAD

**Discussion by airtonix** [rebelot/heirline.nvim#145] [Q&A]
> 🗒️  Use case: 
> 
> 1. double click a buffer tab,
>    - maximises the tab split
> 2. double click a maximised tab
>    - unmaximise tab split

I can see we have `on_click`, but does `on_dbl_click` exist?

**Comment by rebelot** [comment]
As per cookbook documentation: 
`on_click.callback` has the signature `fun(self, minwid, nclicks, button, mods)` (see `:h 'statusline'` description for `@`)
